src/quo/components/wallet/network_routing/view.cljs

```diff
@@ -172,6 +173,16 @@
             [rn/view {:style (style/max-limit-bar-background network-name)}]
             [dashed-line network-name]])]))))
 
+(def ?schema
+  [:=>
+   [:catn
+    [:props
+     [:map {:closed true}
+      [:networks [:maybe :coll-of :any]]
```

Comment:
The `:coll-of` should go into its own vector like
```clj
[:networks [:maybe [:coll-of :any]]]

```
But also, to make sure that both collections and vectors can be passed, would be better to replace `:coll-of` with `:sequential`